### PR TITLE
Improve Android setup

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@edewit @abstractj Could you test the Android setup?

I followed the instructions in Android setup steps and could post a photo on my Facebook

screenshot 2015-03-05 16 01 43

File: Shoot/README.md [hunk 1]
```diff
@@ -122,6 +124,13 @@ $ open platforms/ios/Shoot.xcodeproj/
 
 #### Android setup
 
+```bash
+$ cordova platform add android@master --usegit
```
Comment (edewit) [inline Shoot/README.md]: why would we want to install the master version of the android platform and not the latest version?

Comment (danielpassos) [inline Shoot/README.md]: Because the master has a better support for Gradle for now.

**Comment by edewit** [comment]
Closed in favour of https://github.com/aerogear/aerogear-cordova-oauth2/pull/17

**Pull request closed without merging**